.github/workflows/daily_ete_test.yml

```diff
@@ -33,7 +55,10 @@ jobs:
           dpkg -i /root/packages/allure_2.24.1-1_all.deb
           rm -rf /var/lib/apt/lists/*
       - name: Clone repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          repository: '${{ github.event.inputs.repo_org}}/lmdeploy'
```

Comment:
if it is triggered by `schedule`, then `${{ github.event.inputs.repo_org}}` and `${{github.event.inputs.repo_ref}}` is empty, in which it may fail.

Reply:
I try to fix it. I'm waiting the test workflow my fork is not triggered